server/server.go

```diff
@@ -781,6 +781,13 @@ func (s *Server) updateServerINFO(urls []string) bool {
 	if wasUpdated {
 		s.generateServerInfoJSON()
 	}
+
+	// If notification of server additions is always allowed
```

Comment:
Replace these 2 lines comment with something like: `This means that we should send the INFO protocol even if its content was not updated`

Reply:
Done